test/PackageVersionsTest/InstallerTest.php

```diff
@@ -162,6 +162,7 @@ public function testDumpVersionsClass()
   'foo/bar' => '1.2.3@abc123',
   'baz/tab' => '4.5.6@def456',
   'tar/taz' => '7.8.9@ghi789',
+  'some-replaced/package' => 'self.version@aaabbbcccddd',
```

Comment:
@Jean85 this is what the string would look like. It is fine by me, just need to make sure it isn't a BC break on your side.

Reply:
Nope, I'm good! :+1: 
I've added regression tests on my side: https://github.com/Jean85/pretty-package-versions/commit/44e26e740d2315fb242ce1b96eb3db290299d90e

The only thing is that obviously `self.version` is not getting replaced with the correct one... Is that an issue on your side? It's probably easily fixable with #56

Reply:
I don't have strong feelings, but my (personal) use-case for this library is to create hashes of versions, so that I can decide when to re-generate artifacts. As long as the actual version changes (bit after `@`), I'm good.

Reply:
Since the `replace` feature is used by the `symfony/symfony` package, I think it would be helpful. If both this and #56 get merged, I could attempt a PR to solve this.